apps/src/lib/config/genesis.rs

```diff
@@ -1001,8 +1012,8 @@ pub fn genesis(num_validators: u64) -> Genesis {
         implicit_vp_code_path: vp_implicit_path.into(),
         implicit_vp_sha256: Default::default(),
         max_signatures_per_transaction: 15,
-        epochs_per_year: 525_600, /* seconds in yr (60*60*24*365) div seconds
-                                   * per epoch (60 = min_duration) */
+        epochs_per_year: 365, /* seconds in yr (60*60*24*365) div seconds
```

Comment:
Ah, thanks for pointing this out. I need `epochs_per_year` to equal a small number for the integration tests in order to be able to observe non-zero inflation. This value is directly queried by `calculate_masp_rewards` and used as a quotient in some computations.

For the integration tests, this doesn't seem to be an issue since we manually drive the epochs forward using `MockNode::next_epoch`. And for the E2E tests, this doesn't seem to be an issue since they override the default genesis with `e2e-tests-single-node.toml`. Is there an area where this becomes an issue? Or do you know a better way to achieve the above goal of having observable inflation?

Reply:
I suppose not. I guess in basically all cases, we can override if we need to.